charter.html

```diff
@@ -62,6 +62,7 @@
           <li><a href="#decisions">Decision Policy</a></li>
           <li><a href="#patentpolicy">Patent Policy</a></li>
           <li><a href="#licensing">Licensing</a></li>
+	  <li><a href="#anti-trust">Licensing</a></li>
```

Comment:
Indent looks off.  Have you considered using [tidy](https://github.com/htacg/tidy-html5)?